syntax = "proto3";

package egel_rpc;

service EgelRpc {
    rpc EgelCall(EgelText) returns (EgelText) {}
    rpc EgelBundle(EgelTexts) returns (EgelText) {}
    rpc EgelImport(EgelText) returns (EgelText) {}
    rpc EgelNodeInfo(EgelText) returns (EgelText) {}
}

message EgelText {
    string text = 1;
}

message EgelTexts {
    repeated texts = 1;
}
